js/msp/MSPCodes.js

```diff
@@ -76,9 +76,8 @@ var MSPCodes = {
     MSP_SET_PID_ADVANCED:           95,
     MSP_SENSOR_CONFIG:              96,
     MSP_SET_SENSOR_CONFIG:          97,
-    //MSP_SPECIAL_PARAMETERS:         98, // DEPRECATED
-    //MSP_SET_SPECIAL_PARAMETERS:     99, // DEPRECATED
-    //MSP_IDENT:                      100, // DEPRECTED
+    MSP_CAMERA_CONTROL:             98,
```

Comment:
Yep, will change that back. The MSP commands for the beeper configuration are already reallocated to a free spot. We also need to take care to make new MSP commands when the datacontract changes significantly, the new Battery tab uses the same MSP commands but the datacontract is totally different.